.pylintrc

```diff
@@ -503,7 +503,7 @@ valid-metaclass-classmethod-first-arg=cls
 [DESIGN]
 
 # Maximum number of arguments for function / method.
-max-args=7
+max-args=10
```

Comment:
Can you please help justify this change?

Reply:
there are new command-line options added to the tool. This increase is to accommodate those command-line argument parsing